### PR TITLE
WrapperWidgetUI support externally controlled `expanded` attribute

### PR DESCRIPTION
**Pull request by zbigg**
# Description

Shortcut: https://app.shortcut.com/cartoteam/story/212117/collapsed-state-of-widgets-is-not-saved

WrapperWidgetUI support externally controlled `expanded` attribute, so builder can control it and save in `Widget` state.

If this PR requires a companion in carto-react-template, please also link them both.

## Type of change

- Feature

# Acceptance

Unit-tests added
Demo in https://github.com/CartoDB/cloud-native/pull/5736

# Basic checklist

- Good PR name
- Shortcut link
- Changelog entry
- Just one issue per PR
- GitHub labels
- Proper status & reviewers
- Tests
- Documentation

### CODE REVIEW

**Review by eamador — APPROVED** [review]
LGTM

**Review by bbecquet — APPROVED** [review]

**Review by VictorVelarde — APPROVED** [review]
👍🏻  Let's wait for @borja-munoz confirmation on new behaviour before we merge

**Pull request merged**
